### DISCUSSION TITLE
use init_bundle() for creating new MONAI Bundle

### DISCUSSION THREAD

**Discussion by hw-ju** [Project-MONAI/tutorials#1124] [Q&A]
Hi there!
To create my own MONAI Bundle for a pretrained model, shall I first use `init_bundle` to generate some default configuration files and then modify those files e.g., config files, metadata file? 

The doc for `init_bundle` says it can "Initialise a new bundle directory with some default configuration files and optionally network weights.", but what are the **default configuration files** and what shall users expect to modify in those default files by themselves? 

Besides, `init_bundle` can't automatically fill in entries like `network`, `deterministic_transforms` in the tutorial spleen segmentation [`train.json`](https://github.com/Project-MONAI/tutorials/blob/main/bundle/spleen_segmentation/configs/train.json), correct? 

Could you help me to understand better? Thanks!

**Comment by KumoLiu** [comment]
Hi @hw-ju, thanks for your interest in MONAI bundle!
First, you can start with `init_bundle`, but as you understand, it only initializes a new bundle directory with some default configuration files and optionally network weights. I think you could refer to the CONTRIBUTION GUIDE under the model_zoo repo. 
https://github.com/Project-MONAI/model-zoo/blob/dev/CONTRIBUTING.md
If you have any questions during creating the bundle, please feel free to ask or create a discussion.

Hope it can help you!
Thanks!
